File: DEBERES/Ejemplo 2 Implementacion de Menus y Control de Flujo/menusControldeFlujo.cpp
#include <iostream> 

#include <string> 
using namespace std; 

void ejecutarDeposito(); 

void ejecutarRetiro(); 

void mostrarSaldo(); 

void mostrarError(string mensaje); 
int main() { 

    int opcion; 

    cout << "--- MENU BANCARIO ---" << endl; 

    cout << "1. Deposito" << endl; 

    cout << "2. Retiro" << endl; 

    cout << "3. Mostrar Saldo" << endl; 

    cout << "Elija una opcion: "; 

    cin >> opcion; 

    switch (opcion) { 

        case 1: 

            ejecutarDeposito(); 

            break;  

        case 2: 

            ejecutarRetiro(); 

            break; 

        case 3: 

            mostrarSaldo(); 

            break; 

        default:  

            mostrarError("Opcion no valida"); 

            break; 

    } 
    return 0; 

} 
void ejecutarDeposito() { 

    cout << "Ejecutando deposito... Transaccion exitosa." << endl; 

} 
void ejecutarRetiro() { 

    cout << "Ejecutando retiro... Por favor tome su dinero." << endl; 

} 
void mostrarSaldo() { 

    cout << "Mostrando saldo actual... Su saldo es $100.00" << endl; 

} 
void mostrarError(string mensaje) { 

    cout << "Error: " << mensaje << endl; 

} 

File: DEBERES/Ejemplo 2 Implementacion de Menus y Control de Flujo/menusControldeFlujo.java
import java.util.Scanner; 
public class menusControldeFlujo { 

  

    public static void main(String[] args) { 

        try (Scanner scanner = new Scanner(System.in)) {

            System.out.println("1. Deposito\n2. Retiro\n3. Saldo"); 

            System.out.print("Elija una opcion: "); 

            int opcion = scanner.nextInt(); 

            switch (opcion) {
                case 1 -> ejecutarDeposito();
                case 2 -> ejecutarRetiro();
                case 3 -> mostrarSaldo();
                default -> mostrarError("Opcion no valida");
            }
        }

    } 

    public static void ejecutarDeposito() { System.out.println("Deposito exitoso."); } 

    public static void ejecutarRetiro() { System.out.println("Retiro exitoso."); } 

    public static void mostrarSaldo() { System.out.println("Su saldo es $100.00"); } 

    public static void mostrarError(String mensaje) { System.out.println("Error: " + mensaje); } 

} 

 